zrml/simple-disputes/src/tests.rs

```diff
@@ -1,128 +0,0 @@
-#![cfg(test)]
```

Comment:
After some thought, it was clear that `simple-disputes` tests are a subset of the `PM` markets and the same would apply to `court`. Therefore, all individual pallet tests were transferred to the `PM` pallet.
Since `simple-disputes` is currently the only used "back-end", no dispute mechanism switch was added (https://github.com/zeitgeistpm/zeitgeist/issues/227).